src/style.css

```diff
@@ -0,0 +1,24 @@
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars--1 :nth-child(-n + 1),
+.stars--2 :nth-child(-n + 2),
+.stars--3 :nth-child(-n + 3),
+.stars--4 :nth-child(-n + 4),
+.stars--5 :nth-child(-n + 5) {
+  background: no-repeat url(images/star-active.svg);
+  background-position: 50% 50%;
```

Comment:
You don't need the property here
```suggestion
```

Reply:
Just move `.stars__star` selector above, and you don't need to repeat `background-position` here

Reply:
It doen't work without here